lib/tdf3/src/client/index.ts

```diff
@@ -100,6 +100,7 @@ export interface ClientConfig {
   clientId?: string;
   dpopEnabled?: boolean;
   kasEndpoint?: string;
+  easEndpoint?: string;
```

Comment:
are we still using this? pooey

Reply:
I know we're moving to OIDC, but I couldn't locally test this feature in a browser in development without being able to alter the `accounts` endpoint